ci/build.sh

```diff
@@ -6,9 +6,9 @@ set -e
 
 source $(dirname $0)/_main.sh
 
-runInBuilder "set -e; ./gradlew -p subprojects build ${GRADLE_ARGS} --stacktrace"
-
 # TODO MBS-11769 make single ./gradlew build
+runInBuilder "set -e; ./gradlew -p subprojects build ${GRADLE_ARGS} --stacktrace"
 runInBuilder "set -e; ./gradlew :check ${GRADLE_ARGS} --stacktrace"
+runInBuilder "set -e; ./gradlew :samples:test-runner:build ${GRADLE_ARGS} --stacktrace"
```

Comment:
I've tried to create buildAll as we discussed with @dsvoronin but faced that GRADLE_ARGS do not pass to included builds
So we have to figure out how to share `GRADLE_ARGS` with included builds